src/components/CurrencyInputPanel/index.tsx

```diff
@@ -109,15 +138,29 @@ const LabelRow = styled.div`
   font-size: 0.75rem;
   line-height: 1rem;
   padding: 0 1rem 1rem;
+
   span:hover {
     cursor: pointer;
     color: ${({ theme }) => darken(0.2, theme.deprecated_text2)};
   }
 `
 
-const FiatRow = styled(LabelRow)`
+const FiatRow = styled(LabelRow)<{ phase0Flag: boolean }>`
   justify-content: flex-end;
-  height: 16px;
+  padding: ${({ phase0Flag }) => phase0Flag && '8px 0px'};
```

Comment:
does this work? i thought you need to do the conditional outside? otherwise this would be padding: false right?

Reply:
does padding false just default to 0?